tests/integration/utils.py

```diff
@@ -182,3 +212,48 @@ def cleanup_table(conn, table_name: str):
             cur.close()
         except Exception as e:
             print(f"Error during cleanup: {e}")
+
+def generate_name_variants(name, include_blank = False):
+    lowercase = name.lower()
+    uppercase = name.upper()
+    mixed_case = name.capitalize()
+    in_quotes = '"' + name.upper() + '"'
+    print([lowercase, uppercase, mixed_case, in_quotes])
+    return [lowercase, uppercase, mixed_case, in_quotes]
+
+def generate_select_statement_combos(table, schema = None, database = None):
+    select_statements = []
+    table_variants = generate_name_variants(table)
+
+    if database is not None:
+        database_variants = generate_name_variants(database)
+        schema_variants = generate_name_variants(schema)
+        object_name_combos = product(database_variants, schema_variants, table_variants)
+        for db_name, schema_name, table_name in object_name_combos:
+            select_statements.append(f"SELECT * FROM {db_name}.{schema_name}.{table_name}")
+    else:
+        if schema is not None:
+            schema_variants = generate_name_variants(schema)
+            object_name_combos = product(schema_variants, table_variants)
+            for schema_name, table_name in object_name_combos:
+                select_statements.append(f"SELECT * FROM {schema_name}.{table_name}")
+        else:
+            for table_variant in table_variants:
+                select_statements.append(f"SELECT * FROM {table_variant}")
+
+    return select_statements
+
+def generate_usql_connection_params(account, user, password, role, database = None, schema = None):
```

Comment:
can we use kwargs in `universql_connection` instead? it looks like this is only populating an object with the same keys.